src/components/diff/NotebookDiff.ts

```diff
@@ -126,19 +145,36 @@ export class NotebookDiff extends Panel implements Git.Diff.IDiffWidget {
     if (this._areUnchangedCellsHidden !== v) {
       Private.toggleShowUnchanged(
         this._scroller,
+        this._isConflict,
         this._areUnchangedCellsHidden
       );
       this._areUnchangedCellsHidden = v;
     }
   }
 
+  /**
+   * Helper to determine if a notebook merge should be shown.
+   */
+  private get _isConflict(): boolean {
+    return !!this._model.base;
+  }
+
   /**
    * Promise which fulfills when the widget is ready.
    */
   get ready(): Promise<void> {
     return this._isReady;
   }
 
+  /**
+   * Gets the file contents of a resolved merge conflict,
+   * and rejects if unable to retrieve.
+   */
+  async getResolvedFile(): Promise<string> {
+    // TODO: Implement
+    return Promise.reject('TODO');
```

Comment:
The code used in nbdime to get the resolved notebook is there: https://github.com/jupyter/nbdime/blob/a74b538386d05e3e9c26753ad21faf9ff4d269d7/packages/webapp/src/app/save.ts#L20

You could determine if there are remaining conflict with: `model.conflicts.length > 0`

Reply:
We don't store the nbdime model or the widget  in the Diff class, so to get it into that tunction I can either

a) Create a private var maybe like `this._nbModel` or `this._nbWidget` and call it that way
b) Use `this._scroller.widgets` (type `readonly Widget[]`) and find the correct Widget (I'm assuming its `widgets[0]`)

Which way do you think is better? Option a) is a bit more readable, since we know what the exact widget that we want is. But option b) probably requires less code change

Reply:
I would go for adding a private var `_nbWidget`. The one for the model is not needed as you should be able to get it with `_nbWidget.model`.